src/renderer/modals/FullNode/helpers.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import { ipcRenderer } from "electron";
+import resolveUserDataDirectory from "~/helpers/resolveUserDataDirectory";
+import { setEnvOnAllThreads } from "~/helpers/env";
+import {
+  editSatsStackConfig,
+  stringifySatsStackConfig,
+  parseSatsStackConfig,
+} from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import type { SatsStackConfig } from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import path from "path";
+const lssFileName = "lss.json";
+
+export const saveLSS = async (lssConfig: SatsStackConfig) => {
+  const userDataDirectory = resolveUserDataDirectory();
+  const filePath = path.resolve(userDataDirectory, lssFileName);
+
+  if (filePath) {
+    const configStub = {
+      node: { host: "", username: "", password: "" },
+      accounts: [],
+    };
+    const maybeExistingConfig = (await loadLSS()) || configStub;
+    const updated = editSatsStackConfig(maybeExistingConfig || { accounts: [] }, lssConfig);
```

Comment:
I don't understand why we `|| { accounts: [] }` with the configStub logic just done before.

Reply:
Me neither, probably trying to make flow happy at some point